packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -22,7 +22,7 @@
   width: var(--space-base);
   height: var(--space-base);
   box-sizing: border-box;
-  margin: 0 var(--space-small) 0 0;
+  margin: var(--space-smaller) var(--space-small) 0 0;
```

Comment:
This margin isn't needed, it actually breaks the alignment:
 
![image](https://user-images.githubusercontent.com/39704901/221040734-9b789484-9b1c-4424-8902-ca4db86d8ecf.png)

Reply:
```suggestion
  margin: 0 var(--space-small) 0 0;
```

Reply:
I think it might be conditionally required, because it also looks off when there are multiple children and the margin is provided. LMK if that makes sense?

Reply:
Do you have an example?

Reply:
Hey @chris-at-jobber thanks for the review! This is what I am seeing when trying things out locally.

# Flex Center, Margin Top 0
This is as it currently is in the published version of this package in `master`.

<img width="129" alt="image" src="https://user-images.githubusercontent.com/24949738/221099710-63b703ee-50d0-4dd8-9627-60a07c970ecb.png">

# Flex Start, Margin Top Space-Smaller
This is what it looks like with the proposed change in this PR

<img width="131" alt="image" src="https://user-images.githubusercontent.com/24949738/221100039-7b623524-f8e6-4be0-bebd-18adabb66ad3.png">

# Flex Start, Margin Top 0
This is what it looks like with the suggested change:

<img width="127" alt="image" src="https://user-images.githubusercontent.com/24949738/221099813-c874e729-f66c-4cdc-97ab-5f234e61e84b.png">

In conclusion, none of these seem pixel-perfect to me, but I am definitely not a designer and am very open to other suggestions/feedback on how to get around this. In the meantime, I have updated my work that needs these changes to use a workaround (basically I am doing the same thing as what I am proposing in this PR, except only scoped to the component/UI needing this change). Once we have a path forward here, I can go back and remove my temporary fix.

Thanks in advance for feedback!

Reply:
```suggestion
  margin: var(--space-smallest) var(--space-small) 0 0;
```

Reply:
Sweet - change made!